src/atc/orchestrators/ehjson2delta/EhJsonToDeltaTransformer.py

```diff
@@ -0,0 +1,58 @@
+from pyspark.sql import DataFrame
+from pyspark.sql import functions as f
+
+from atc.delta import DeltaHandle
+from atc.eh import EventHubCaptureExtractor
+from atc.etl import Transformer
+
+
+class EhJsonToDeltaTransformer(Transformer):
+    def __init__(self, eh: EventHubCaptureExtractor, target_dh: DeltaHandle):
+        super().__init__()
+        self.eh = eh
+        self.target_dh = target_dh
+
+    def process(self, df: DataFrame) -> DataFrame:
+        # fist add the pdate column that is composed of the y, m, d and h columns:
```

Comment:
mispelling "first"

Reply:
it was  already taken care of here https://github.com/atc-net/atc-dataplatform/blob/main/src/atc/eh/EventHubCaptureExtractor.py#L73